css/index.css

```diff
@@ -135,6 +135,14 @@ html,
 body {
   height: 100%;
 }
+.a-font {
+  text-decoration: none;
+  font-size: 0.95rem;
+}
+.p-font {
```

Comment:
Here is what I was talking about.. the mixins you created were included in here, your compiled index.css 😄